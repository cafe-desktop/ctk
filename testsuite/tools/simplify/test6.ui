<!-- non-canonical property name works -->
<interface>
  <object class="CtkWindow">
    <property name="default_width">-1</property>
  </object>
</interface>
