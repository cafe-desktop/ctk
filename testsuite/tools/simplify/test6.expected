<!-- non-canonical property name works -->
<interface>
  <object class="CtkWindow">
  </object>
</interface>
